engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/confluent/client/CachedConfluentSchemaRegistryClient.scala

```diff
@@ -38,6 +40,18 @@ class CachedConfluentSchemaRegistryClient(val client: CSchemaRegistryClient, sch
       }
     }
 
+
+  override def getAllTopics: Validated[SchemaRegistryError, List[String]] = handleClientError {
+    val pattern = "(.*)-value".r
```

Comment:
Nice trick ;)

Reply:
"when you have a problem and solve it with regexp..." ;)